client/README.md

```diff
@@ -21,105 +21,100 @@ Full docs can be found at https://qiskit-extensions.github.io/qiskit-serverless/
-   # we want to sample in parallel.
-   # Let's use `get_arguments` funciton
-   # to access all program arguments
-   arguments = get_arguments()
-   circuits = arguments.get("circuits", [])
-
-   # 3. run our functions in a loop
-   # and get execution references back
-   function_references = [
-       distributed_sample(circuit)
-       for circuit in circuits
-   ]
-
-   # 4. `get` function will collect all
-   # results from distributed functions
-   collected_results = get(function_references)
-
-   # 5. `save_result` will save results of program execution
-   # so we can access it later
-   save_result({
-       "quasi_dists": collected_results
-   })
+from qiskit_serverless import distribute_task, get, get_arguments, save_result
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.random import random_circuit
+from qiskit.primitives import StatevectorSampler as Sampler
+from qiskit.quantum_info import SparsePauliOp
+
+# 1. let's annotate out function to convert it
+# to distributed async function
+# using `distribute_task` decorator
+@distribute_task()
+def distributed_sample(circuit: QuantumCircuit):
+    """Calculates quasi dists as a distributed function."""
+    return Sampler().run([(circuit)]).result()[0].data.meas.get_counts()
+
+# 2. our program will have one arguments
+# `circuits` which will store list of circuits
+# we want to sample in parallel.
+# Let's use `get_arguments` funciton
+# to access all program arguments
+arguments = get_arguments()
+circuits = arguments.get("circuits", [])
+
+# 3. run our functions in a loop
+# and get execution references back
+function_references = [
+    distributed_sample(circuit)
+    for circuit in circuits
+]
+
+# 4. `get` function will collect all
+# results from distributed functions
+collected_results = get(function_references)
+
+# 5. `save_result` will save results of program execution
+# so we can access it later
+save_result({
+    "quasi_dists": collected_results
+})
 ```
  
 
-### Step 2: run pattern
+### Step 2: run function
 
 ```python
-   from qiskit_serverless import ServerlessProvider, QiskitPattern
-   from qiskit.circuit.random import random_circuit
-
-   serverless = ServerlessProvider(
-       username="<USERNAME>", 
-       password="<PASSWORD>",
-       host="<GATEWAY_ADDRESS>",
-   )
-
-   # create program
-   program = QiskitPattern(
-       title="Quickstart",
-       entrypoint="pattern.py",
-       working_dir="./src"
-   )
-
-   # create inputs to our program
-   circuits = []
-   for _ in range(3):
-       circuit = random_circuit(3, 2)
-       circuit.measure_all()
-       circuits.append(circuit)
-
-   # run program
-   job = serverless.run(
-       program=program,
-       arguments={
-           "circuits": circuits
-       }
-   )
+from qiskit_serverless import ServerlessClient, QiskitFunction
+from qiskit.circuit.random import random_circuit
+
+client = ServerlessClient(
+    token="awesome_token", 
+    host="http://localhost:59994",
```

Comment:
Is it port `599994` the default one, @akihikokuroda ? Just to confirm 👍

Reply:
oops.  I was going to change them before checkin.  I'll fix it.